cockroachcloud/billing-management.md

```diff
@@ -7,7 +7,7 @@ docs_area:
 
 The **Billing** page contains an overview of your charges and the payment details on file for your {{ site.data.products.db }} organization. To view the **Billing** page, [log in](https://cockroachlabs.cloud/) and click **Billing**.
 
-[Console Admins](console-access-management.html#console-admin) can set up and manage billing for the organization.
+[Org admins](authorization.html#org-administrator-legacy) can set up and manage billing for the organization.
```

Comment:
```suggestion
[Org admins (legacy)](authorization.html#org-administrator-legacy) can set up and manage billing for the organization.
```

Reply:
I'm not sure about adding the 'legacy' role to all usages of "Org administrator" throughout the docs at this point. If people click through to the definition of the role, they'll see the legacy label. For one thing, it's just unweildy language-wise, and secondly, it only applies after opt-in (and I wasn't going to put the opt-in note on every page throughout the docs where the org admin role is mentioned, that would be a lot of note noise.